qiskit_experiments/randomized_benchmarking/rb_experiment.py

```diff
@@ -162,3 +168,36 @@ def _generate_circuit(
                 rb_circ.measure_all()
                 circuits.append(rb_circ)
         return circuits
+
+    def run(
+        self,
+        backend: Backend,
+        analysis: bool = True,
+        experiment_data: Optional[ExperimentData] = None,
+        **run_options,
+    ) -> ExperimentData:
+        """Run an experiment and perform analysis.
+
+        Args:
+            backend: The backend to run the experiment on.
+            analysis: If True run analysis on the experiment data.
+            experiment_data: Optional, add results to existing
+                experiment data. If None a new ExperimentData object will be
+                returned.
+            run_options: backend runtime options used for circuit execution.
+
+        Returns:
+            The experiment data object.
+        """
+        error_dict = RBUtils.get_error_dict_from_backend(backend, self.physical_qubits)
```

Comment:
You need to check if `error_dict` is `None` before overriding. User may manually set the option before running experiment.